source/System.cpp

```diff
@@ -538,6 +539,15 @@ void System::UpdateSystem(const Set<System> &systems, const set<double> &neighbo
 
 
 
+void System::UpdateObjectVisibilities(const Ship *flagship)
```

Comment:
Since we don't need to update the visibilities when entering a system, this method isn't useful anymore. The same can be said about the GameData function.